app/controllers/consolidations/ManageMucrController.scala

```diff
@@ -44,7 +44,7 @@ class ManageMucrController @Inject()(
 
   def displayPage(): Action[AnyContent] = (authenticate andThen ileQueryFeatureEnabled andThen getJourney(JourneyType.ASSOCIATE_UCR)) {
     implicit request =>
-      if (request.cache.queryUcr.map(_.is(UcrType.Ducr)).getOrElse(throw InvalidFeatureStateException)) {
+      if (request.cache.queryUcr.map(!_.is(UcrType.Mucr)).getOrElse(throw InvalidFeatureStateException)) {
```

Comment:
Probably will be easier to read when is method could take more than one UcrType and we can invoke it `is(Ducr, DucrPart)`

Reply:
Or maybe implement `isNot(_)` method instead?

Reply:
Yeah, I thought about it, but is not is not saying what is allowed, not sure. `is(Ducr, DucrParts)` say more than `isNot(Mucr)`

In general this is just a suggestion, non blocking :)